### ISSUE TITLE
The Post Navigation Link does not have a clickable button

### ISSUE THREAD

**Issue by ildomandatore** [WordPress/gutenberg#62197]
### Description

The Post Navigation Link does not have the clickable button, but only the link has the clickable element. Could you make the entire button pressable and not just its text?

### Step-by-step reproduction instructions

Hover over the button, you will notice that the cursor does not appear and we cannot go to the chosen post. Instead, pressing on the text works, but could create navigation difficulties for those who do not press on the text, but on the button.

### Screenshots, screen recording, code snippet

https://imgur.com/a/ZtyBYJW

### Environment info

wordpress version 6.5.3, theme Twenty Twenty-Four, block Post Navigation Link, Apparence -> Customize -> Templetes - > Single Posts, Firefox 126.0.1 desktop.

### Please confirm that you have searched existing issues in the repo.

Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

Yes

**Comment by carolinan** [comment]
Hi
The reason why this block is different from the button block, is that by design, it is intended to be a text or arrow _combined with a link_. This block has an option to choose which part of the block that is linked, the post title, the custom label, or both.
This allows theme designers to create different styles that can't be done with the button block.

I do understand what it is you want the block to do, but it is not a bug, that is why I changed it to an enhancement.

It may be worth exploring if instead, the **button block**, which behaves the way that you want, could have the option to link to the next or previous post.

Developers can also help you achieve what you are asking for, by adding the next or previous link to the button block with the block bindings API.

**Comment by ildomandatore** [comment]
It actually has the option for background color.
https://imgur.com/a/tzhT7eb

Not even the buttons have an option to go to the previous and next page.
https://imgur.com/a/msKGD93
https://imgur.com/a/kaLElCn

The button links to the previous and next pages if I enter the urls manually. But I want to avoid this extra process. It's strange that no one pointed out that there was no option to create buttons for Post Navigation Links.

Other alternatives to solve the problem? In the meantime I'm looking to see if there is a plugin.

**Comment by carolinan** [comment]
Yes I know the button does not have the option, that is why I said it might be worth exploring if the option could be added.

**Comment by ildomandatore** [comment]
If I understood correctly, am I waiting for the idea I gave to be taken into consideration by Wordpress?
